app/components/ClueMode.tsx

```diff
@@ -902,6 +907,53 @@ export const ClueMode = ({ state, ...props }: ClueModeProps) => {
               }
             />
           </div>
+          <div>
+            <h3>Enumerations</h3>
+            <div>
+              <ButtonAsLink
+                text="Add Enumerations"
+                onClick={() => {
+                  props.dispatch({
+                    type: 'ADDENUMERATIONS',
+                  } as AddEnumerationsAction);
+                  showSnackbar(
+                    <>
+                      Enumerations added!&nbsp;&nbsp;
+                      <ButtonAsLink
+                        text="Undo"
```

Comment:
I think we need to somehow improve the readability of "Undo" here. Unfortunately the link colors are chosen to appear against a white bg for normal mode and a dark bg for dark mode, and the snackbar bg is the exact opposite! I looked through the color vars we have defined and don't see a great option to use for this. I honestly hate to add yet another color - if anything I'd love to try to slim down our massive list - do you think using `var(--snackbar-text)` for the undo button would be too subtle (i.e. matching the "Enumerations added!" text)? If we go that route we should definitely add a text-decoration:underline even to the non-hover state to give a hint that the undo is clickable.

Reply:
Yeah I see what you mean, the colors really don't work as-is, especially in dark mode!

<img width="265" alt="Screenshot 2024-05-04 at 9 55 08 PM" src="https://github.com/crosshare-org/crosshare/assets/1813467/f11f3dd8-a3f8-43ef-8a06-2b1d095b3c58">

So if it used `var(--snackbar-text)`, would the hover and active state colors be disabled? I've seen like dotted underline that changes to solid on hover.

<img width="241" alt="Screenshot 2024-05-04 at 10 09 02 PM" src="https://github.com/crosshare-org/crosshare/assets/1813467/3d09554c-6b9b-4c26-8b89-d6f2941855aa">

I'm happy to go that route, though I _do_ sort of think it's a little more subtle than you'd ideally want. 

Another thought, perhaps crackpot: if _any_ node can go in the snackbar, you'd almost want the snackbar to act like an entire context provider for the _reverse_ (non-active) theme. And since both themes are already in the stylesheet, there could be a class to select the reverse theme, something like:

```
body.light-mode, body.dark-mode .reverse-theme {
  ${colorThemeString({ ... })
}
```

Is that too weird? I'm happy to go with whatever for this!

Reply:
Ah yeah I think you're right, might need to add the theme string again!

Reply:
We could use the same reverse trick for `ToolTipText` as well, I think, which might help to justify that further.